doc/kazoo/cluster-guide.md

```diff
@@ -1,9 +1,311 @@
+Returns  
+```{"_id":"bigcouch@couch1.z100.somedomain.com","_rev":"3-b13d076f367df4d0c52b236e654b836c"}```  
+
+Now add the zone to this bigcouch server and cluster together the other servers.
+```
+curl -X PUT couch1.z100.somedomain.com:5986/nodes/bigcouch@couch1.z100.somedomain.com -d '{"_rev":3-b13d076f367df4d0c52b236e654b836c", "zone":"z100"}'
+curl -X PUT couch1.z100.somedomain.com:5986/nodes/bigcouch@couch2.z100.somedomain.com -d '{"zone":"z100"}'
+curl -X PUT couch1.z100.somedomain.com:5986/nodes/bigcouch@couch3.z100.somedomain.com -d '{"zone":"z100"}'
+curl -X PUT couch1.z100.somedomain.com:5986/nodes/bigcouch@couch1.z100.somedomain.com -d '{"zone":"z200"}'
+curl -X PUT couch1.z100.somedomain.com:5986/nodes/bigcouch@couch2.z100.somedomain.com -d '{"zone":"z200"}'
+curl -X PUT couch1.z100.somedomain.com:5986/nodes/bigcouch@couch3.z100.somedomain.com -d '{"zone":"z200"}'
+```
+Verify cluster:  
+`curl http://couch1.z100.somedomain.com:5984/_membership`  
+Should return:  
+```
+{"all_nodes":["bigcouch@couch3.z200.somedomain.com","bigcouch@couch2.z200.somedomain.com","bigcouch@couch1.z200.somedomain.com",
+"bigcouch@couch3.z100.somedomain.com","bigcouch@couch2.z100.somedomain.com","bigcouch@couch1.z100.somedomain.com"],
+
+"cluster_nodes":["bigcouch@couch3.z200.somedomain.com","bigcouch@couch2.z200.somedomain.com","bigcouch@couch1.z200.somedomain.com",
+"bigcouch@couch3.z100.somedomain.com"","bigcouch@couch2.z100.somedomain.com","bigcouch@couch1.z100.somedomain.com"]}
+```
+You can do that on each server to verify they all have the same configuration.
+
+To verify zone configuration on each document on each server.
+```
+curl http://couch1.z100.somedomain.com:5986/nodes/bigcouch@couch1.z100.somedomain.com
+curl http://couch1.z100.somedomain.com:5986/nodes/bigcouch@couch2.z100.somedomain.com
+...
+```
+
+etc.  
+### Kazoo Config
+
+After installing kazoo in both zones edit `/etc/kazoo/core/config.ini`.  This is exactly the same on both.  
+```
+; section are between [] = [section]
+; key = value
+; to comment add ";" in front of the line
+;[amqp]
+;uri = "amqp://guest:guest@127.0.0.1:5672"
+
+[zone]
+name = "z100"
+amqp_uri = "amqp://guest:guest@10.100.30.1"
+
+[zone]
+name = "z200"
+amqp_uri = "amqp://guest:guest@10.200.30.1"
+
+[bigcouch]
+compact_automatically = true
+cookie = change_me
+ip = "127.0.0.1"
+port = 15984
+; username = "kazoo"
+; password = "supermegaexcellenttelephonyplatform"
+admin_port = 15986
+
+[whistle_apps]
+cookie = change_me
+
+[kazoo_apps]
+cookie = change_me
+zone = "z100"
+host = "kazoo1.z100.somedomain.com"
+
+[kazoo_apps]
+cookie = change_me
+zone = "z200"
+host = "kazoo1.z200.somedomain.com"
+
+[ecallmgr]
+cookie = change_me
+zone = "z100"
+host = "kazoo1.z100.somedomain.com"
+
+[ecallmgr]
+cookie = change_me
+zone = "z200"
+host = "kazoo1.z200.somedomain.com"
+
+[log]
+syslog = info
+console = notice
+file = error
+```
+
+### Assign Freeswitch to Ecallmgr zones
```

Comment:
we don't recommend to edit documents in futton. use `sup -n ecallmgr ecallmgr_maintenance` commands to add freeswitch nodes